tests/snapshots.rs

```diff
@@ -225,15 +225,16 @@ fn write_output_spv(
                 entry_point: ep.name.clone(),
                 shader_stage: ep.stage,
             };
-            let spv = spv::write_vec(module, info, &options, Some(&pipeline_options)).unwrap();
+            let spv = spv::write_vec(module, info, &options, Some(&pipeline_options))
+                .expect("SPV EP write failed");
             let dis = rspirv::dr::load_words(spv)
                 .expect("Produced invalid SPIR-V")
                 .disassemble();
             let path = format!("spv/{}.{}.spvasm", file_name, ep.name);
             fs::write(destination.join(path), dis).unwrap();
         }
     } else {
-        let spv = spv::write_vec(module, info, &options, None).unwrap();
+        let spv = spv::write_vec(module, info, &options, None).expect("SPV write failed");
```

Comment:
Oh, these changes might conflict with #1427. Oh well!

Reply:
I'll remove it